main/views.py

```diff
@@ -1,3 +1,6 @@
 from django.shortcuts import render
 
 # Create your views here.
+
+def index(request):
+    return render(request, 'main\index.html')
```

Comment:
index.html doesn't work because the path is relative to the app's template folder. templates need to be kept in a sub folder with the same name as the app hence "main/index.html"